WaterSmartMeter/ngsild-payloads/consumer.json

```diff
@@ -0,0 +1,152 @@
+    },
+    "persistenceFlowDuration": {
+        "type": "Property",
+        "value": 3,
+        "unitCode": "HUR",
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "acquisitionStageFailure": {
+        "type": "Property",
+        "value": 0,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmFlowPersistence": {
+        "type": "Property",
+        "value": 0,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmInProgressAlarms": {
+        "type": "Property",
+        "value": 1,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmMetrology": {
+        "type": "Property",
+        "value": 1,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmStopsLeaks": {
+        "type": "Property",
+        "value": 1,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmSystem": {
+        "type": "Property",
+        "value": 1,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmTamper": {
+        "type": "Property",
+        "value": 0,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "alarmWaterQuality": {
+        "type": "Property",
+        "value": 0,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "moduleTampered": {
+        "type": "Property",
+        "value": 1,
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "location": {
+        "type": "GeoProperty",
+        "value": {
+            "type": "Point",
+            "coordinates": [
+                24.30623,
+                60.07966
+            ]
+        }
+    },
+    "@context": [
+        "https://schema.lab.fiware.org/ld/context.jsonld"
```

Comment:
we should provide the other context also, no?

Reply:
yes, but after mergin in master to get the right url